scripts/globals/abilities/third_eye.lua

```diff
@@ -17,9 +17,9 @@ function onAbilityCheck(player, target, ability)
 end
 
 function onUseAbility(player, target, ability)
-    if player:hasStatusEffect(tpz.effect.COPY_IMAGE) then
+    if (player:hasStatusEffect(tpz.effect.COPY_IMAGE) or player:hasStatusEffect(tpz.effect.BLINK)) then
```

Comment:
Content looks good, could you just please remove the containing parentheses for styling?